Casks/b/bambu-suite.rb

```diff
@@ -0,0 +1,32 @@
+cask "bambu-suite" do
```

Comment:
```suggestion
cask "bambusuite" do
```

Based on our [`token reference`](https://docs.brew.sh/Cask-Cookbook#token-reference) the token should be updated, you'll also need to adjusted the filename to match.

Reply:
As you wish :)